doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -308,12 +310,18 @@ join_expr ::=
 	| select_pred join_type 'JOIN' table_ref ( 'USING' '(' ( ( col_ref ) ( ( ',' col_ref ) )* ) ')' | 'ON' expression ) select_post
 join_type ::=
     ( 'FULL' ( 'OUTER' |  ) | 'LEFT' ( 'OUTER' |  ) | 'RIGHT' ( 'OUTER' |  ) | 'INNER' | )
+jsonb_agg ::=
+    'jsonb_agg' '(' expression ( 'ORDER' 'BY' col_ref ( 'ASC' | 'DESC' )? ( ',' col_ref ( 'ASC' | 'DESC' )? )* )? ')'
+jsonb_object_agg ::=
+    'jsonb_object_agg' ( ( '(' key ':' value (',' key ':' value )?)?  ( 'ORDER' 'BY' col_ref ( 'ASC' | 'DESC' )? ( ',' col_ref ( 'ASC' | 'DESC' )? )* )? ')' )
```

Comment:
This doesn't take a json object delimited with `:`, it takes exactly two arguments:

```
jsonb_object_agg ::= 'jsonb_object_agg' '(' keys ',' values ( 'ORDER' 'BY' col_ref ( 'ASC' | 'DESC' )? ( ',' col_ref ( 'ASC' | 'DESC' )? )* )? ')' ')'
```